oxidation/generate_test_data/protocol/user.py

```diff
@@ -52,6 +63,28 @@
 serializer.req_loads(serialized)
 display("user_create_req", serialized, [])
 
+serialized = serializer.req_dumps(
+    {
+        "cmd": "user_create",
+        "user_certificate": b"foobar",
+        "device_certificate": b"foobar",
+        "redacted_user_certificate": b"foobar",
+        "redacted_device_certificate": None,
+    }
+)
+display("user_create_req_redacted_device_certificate_none", serialized, [])
```

Comment:
no `serializer.rep_loads(serialized)` ?

Reply:
I can't because it will fail

Reply:
good point ! 👍 (but it would be better to add comment to make this explicit)